wiki/Main_Page/en.md

```diff
@@ -29,42 +35,80 @@ Welcome to the osu! wiki, a project containing a wide range of osu! related info
+
 [Submission](/wiki/Submission) • [Modding](/wiki/Modding) • [Ranking procedure](/wiki/Beatmap_ranking_procedure) • [Mappers' Guild](/wiki/Mappers_Guild) • [Project Loved](/wiki/Project_Loved)
 
 [Ranking criteria](/wiki/Ranking_Criteria): [osu!](/wiki/Ranking_Criteria/osu!) • [osu!taiko](/wiki/Ranking_Criteria/osu!taiko) • [osu!catch](/wiki/Ranking_Criteria/osu!catch) • [osu!mania](/wiki/Ranking_Criteria/osu!mania)
 
 </div>
 <div class="wiki-main-page-panel">
 
-# Community
+# Rules and legalese
+
+Read about procedures and regulations that keep osu! and its community safe, from music licensing and media guidelines to chat and forum rules.
+
+[Rules](/wiki/Rules) • [Contributor Code of Conduct](/wiki/Contributor_Code_of_Conduct) • [Code of Conduct for modding and mapping](wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping)
+
+Media regulations: [Content usage guidelines](/wiki/Rules/Content_Usage_Guidelines) • [Explicit content](/wiki/Rules/Explicit_Content) • [Song content rules](/wiki/Rules/Song_Content_Rules) • [Visual content considerations](/wiki/Rules/Visual_Content_Considerations)
+
+[Legal pages](/wiki/Legal): [Copyright (DMCA)](/wiki/Legal/Copyright) • [Music licensing](/wiki/Legal/Music_Licensing) • [Privacy](/wiki/Legal/Privacy) • [Terms of Service](/wiki/Legal/Terms)
+
+</div>
+<div class="wiki-main-page-panel">
+
+# Help
 
-[Tournaments](/wiki/Tournaments) • [Skinning](/wiki/Skinning) • [Projects](/wiki/Projects) • [Guides](/wiki/Guides) • [osu!dev Discord server](/wiki/osu!dev_Discord_server) • [How you can help](/wiki/How_You_Can_Help!) • [Glossary](/wiki/Glossary)
+Answers to frequent questions and solutions to common issues live here. Learn how to deal with network issues, or combat bad behaviour in the community.
+
+[Help centre](/wiki/Help_Centre) • [Performance troubleshooting](/wiki/Performance_Troubleshooting) • [Account support team](wiki/People/The_Team/Account_support_team) • [Help forum](https://osu.ppy.sh/forum/5)
+
+Profile restrictions: [Silence](/wiki/Silence) • [Account restrictions](/wiki/Help_Centre/Account_Restrictions)
+
+Do your part: [How and what to help with](/wiki/How_you_can_help!) • [Report bad behaviour](/wiki/Reporting_Bad_Behaviour) • [Report abuse](/wiki/Reporting_Bad_Behaviour/Abuse) • [Report cheating](/wiki/Reporting_Bad_Behaviour/Handling_Foul_Play)
+
+</div>
+<div class="wiki-main-page-panel">
+
+# Community and projects
+
+Social interaction is one of main things that fuel osu! players all around the world. Explore the activities and contests, get in touch with others, or organize the next big event!
+
+[Tournaments](/wiki/Tournaments) • [Featured Artists](/wiki/Featured_Artists) • [Beatmap Spotlights](/wiki/Beatmap_Spotlights) • [Contests](/wiki/Contests) • [Community mentorship program](wiki/Community_Mentorship_Program)
```

Comment:
It feels a little off for Featured Artists to be in this section. It's not something that users can directly get involved with - it's more of an official project. You might want to move it to the project section, or give it its own section directly.

Reply:
whoops, yes.